### PR TITLE
Registration Page

### LINKED ISSUE #5: Add registration page

### PR DESCRIPTION
**Pull request by n-aggarwal**

### CODE REVIEW

File: views/register.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        // Handle login error, display an error message, etc.
+      });
+  };
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.header}>Register Page</Text>
+      <KeyboardAwareScrollView>
+        <TextInput
+          placeholder="Email"
```
Comment (ananafrida) [inline views/register.js]: could you please change it to "username"?

File: views/register.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+} from "react-native";
+import { KeyboardAwareScrollView } from "react-native-keyboard-aware-scroll-view";
+
+import { createStackNavigator } from "@react-navigation/stack";
+import { NavigationContainer } from "@react-navigation/native";
+
+export default function LoginPage({ navigation }) {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [username, setUsername] = useState("");
```
Comment (ananafrida) [inline views/register.js:23]: ig you flipped the role of username and email but doesn't impact the functionality at all. So you don't have to change unless you want to

**Review by ananafrida — CHANGES_REQUESTED** [review]
placeholder change is needed

File: views/register.js (hunk 2 above)
Comment (n-aggarwal) [reply views/register.js:23]: I will leave it as it is for the time being.

**Comment by n-aggarwal** [comment]
I have made the appropriate changes.

**Comment by ananafrida** [comment]
> I have made the appropriate changes.

you are supposed to rerequest review -_-

**Review by ananafrida — APPROVED** [review]

**Comment by n-aggarwal** [comment]
>you are supposed to rerequest review -_-

I thought I did!

**Pull request merged**